docker/dev/docker.env

```diff
@@ -2,6 +2,7 @@ AWS_ACCESS_KEY_ID=x
 AWS_SECRET_ACCESS_KEY=x
 AWS_DEFAULT_REGION=us-east-1
 AWS_ACCOUNT_ID=x
+CHALLENGE_PK=1
```

Comment:
Please change the env variable name to something more meaningful.

Reply:
I think we should not change it now as it'll involve changes at various places for e.g. in deployment script, production, staging env variables files etc.